extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/cors/CORSSecurityTestCase.java

```diff
@@ -18,7 +18,7 @@
 import io.quarkus.vertx.http.security.TestIdentityController;
 import io.quarkus.vertx.http.security.TestIdentityProvider;
 
-public class CoresSecurityTestCase {
+public class CORSSecurityTestCase {
```

Comment:
@Postremus thanks for spotting the typo, it is a minor issue but lets keep the original naming pattern, `CorsSecurityTestCase`

Reply:
@sberyozkin The other test classes in this package are also named CORSXXXXTestCase, which is why I went for this name.